@xen-orchestra/lite/src/components/Table.tsx

```diff
@@ -19,55 +52,318 @@ interface ParentState {}
 
 interface State {}
 
-interface Props {
-  collection: Item[] | undefined
-  columns: Column<any>[]
-  placeholder?: JSX.Element
+interface StatePagination {}
+
+type Props = IsSelectable &
+  RouteComponentProps & {
+    collection: Item[] | undefined
+    columns: Column<any>[]
+    dataType?: string
+    stateUrlParam: string
+  }
+
+interface PropsPagination {
+  dataType?: string
+  nItemsOnThePage: number
```

Comment:
`nItemsPerPage` is different than `nItemsOnThePage`.

In case we are on the last page, we can have `nItemsPerPage === 20` and `nItemsOnThePage === 5`